deployer/debug.py

```diff
@@ -120,18 +121,8 @@ def user_logs(
         subprocess.check_call(cmd)
 
 
-@app.command()
-def exec_homes_shell(
-    cluster_name: str = typer.Argument(..., help="Name of cluster to operate on"),
-    hub_name: str = typer.Argument(..., help="Name of hub to operate on"),
-):
-    """
-    Pop an interactive shell with the home directories of the given hub mounted on /home
-    """
-    # Name pod to include hub name so it is displayed as part of the default prompt
-    # This makes sure we don't end up deleting the wrong thing
-    pod_name = f"{cluster_name}-{hub_name}-shell"
-    pod = {
+def generate_pod_definition_with_home_mount(pod_name):
```

Comment:
I think we can't share pod definition, because the `cp` should be run in a container that's running as uid 1000, while exec-homes-shell probably as `root`. Otherwise the copies end up as owned by root and cause problems - see https://github.com/yuvipanda/prometheus-dirsize-exporter/issues/5.

Reply:
Yeah, definitely copied as root....